docs/src/use/configure/migration-guide.md

```diff
@@ -114,6 +113,99 @@ export default [
 ];
 ```
 
+### Processors
+
+In eslintrc files, processors had to be defined in a plugin, and then referenced by name in the configuration. Processors with a name in the special form `.xxx`  were also automatically configured for any files with the extension `xxx`.
+
+In flat config files, processors can still be referenced from plugins by their name, but they can now also be inserted directly into the configuration. Processors will _never_ be automatically configured, and must be explicitly set in the configuration.
+
+As an example with a custom plugin with processors:
+
+```javascript
+// node_modules/eslint-plugin-somePlugin/index.js
```

Comment:
Since npm package names are not allowed to contain uppercase letters (per https://docs.npmjs.com/cli/v9/configuring-npm/package-json#name), would `eslint-plugin-some-plugin` be a better name?